I/ImageMagick/ImageMagick@7/bundled/patches/urlmon.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile.in b/Makefile.in
+index 182511d3e..daddc466f 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -257,7 +257,7 @@ TESTS = $(TESTS_TESTS) $(am__EXEEXT_5)
+ XFAIL_TESTS = $(am__EXEEXT_6) $(am__EXEEXT_6)
+ check_PROGRAMS = $(am__EXEEXT_2) $(am__EXEEXT_4)
+ @WIN32_NATIVE_BUILD_TRUE@am__append_1 = -lurlmon
+-@WIN32_NATIVE_BUILD_TRUE@am__append_2 = -lws2_32
++@WIN32_NATIVE_BUILD_TRUE@am__append_2 = -lws2_32 -lurlmon
```

Comment:
Upstreamed in https://github.com/ImageMagick/ImageMagick/pull/6032.  It was also quickly merged, so next version of ImageMagick will need to remove this patch.